internal/service/encryptionatrest/resource_encryption_at_rest_migration_test.go

```diff
@@ -79,8 +82,17 @@ func TestAccMigrationAdvRS_EncryptionAtRest_WithRole_basicAWS(t *testing.T) {
 		CheckDestroy: testAccCheckMongoDBAtlasEncryptionAtRestDestroy,
 		Steps: []resource.TestStep{
 			{
-				ExternalProviders: mig.ExternalProvidersWithAWS("5.1.0"),
-				Config:            testAccMongoDBAtlasEncryptionAtRestConfigAwsKmsWithRole(awsKms.Region, accessKeyID, secretKey, projectID, policyName, roleName, false, &awsKms),
+				ExternalProviders: map[string]resource.ExternalProvider{
```

Comment:
i would keep the new way: ExternalProviders: mig.ExternalProvidersWithAWS("5.1.0"),